ncm-metaconfig/src/main/metaconfig/rsyslog/inputs.tt

```diff
@@ -1,4 +1,4 @@
 [%- FOREACH input IN CCM.contents.input.pairs %]
 [% # One input-type per named input
-      INCLUDE 'metaconfig/rsyslog/input.tt' name=input.key type=input.value.keys.0 params=input.value.values.0 -%]
```

Comment:
if name is not passed, why don't you just remove it in the input.tt?

Reply:
Good point. Some input modules do have a name set, so that's why I chose to keep it in input.tt. For example, imtcp may have a Name. Hmm, but then it's capitalised, so I guess if it's set, it'll be in the dict that is walked?